--- PKGBUILD	2022-05-08 11:06:40.234864629 +0800
+++ PKGBUILD2	2022-05-08 12:04:35.998004013 +0800
@@ -62,7 +62,7 @@
 #  39. Generic-x86-64-v4 (GENERIC_CPU4)
 #  40. Intel-Native optimizations autodetected by GCC (MNATIVE_INTEL)
 #  41. AMD-Native optimizations autodetected by GCC (MNATIVE_AMD)
-: "${_subarch:=""}"
+: "${_subarch:="13"}"
 
 # Use the current kernel's .config file
 # Enabling this option will use the .config of the RUNNING kernel rather than
@@ -85,7 +85,7 @@ _major=5.17
 _minor=5
 _srcname=linux-${_major}
 _clr=${_major}.4-1139
-pkgbase=linux-clear
+pkgbase=linux-clear-zen3
 pkgver=${_major}.${_minor}
 pkgrel=1
 pkgdesc='Clear Linux'
@@ -296,7 +296,7 @@ build() {
 }
 
 _package() {
-    pkgdesc="The $pkgdesc kernel and modules"
+    pkgdesc="The $pkgdesc kernel and modules for zen3 architecture"
     depends=('coreutils' 'kmod' 'initramfs')
     optdepends=('wireless-regdb: to set the correct wireless channels of your country'
                 'linux-firmware: firmware images needed for some devices'
@@ -326,7 +326,7 @@ _package() {
 }
 
 _package-headers() {
-    pkgdesc="Headers and scripts for building modules for the $pkgdesc kernel"
+    pkgdesc="Headers and scripts for building modules for the $pkgdesc kernel for zen3 architecture"
     depends=(pahole)
 
     cd ${_srcname}
